### PR TITLE
Add some terraform specific guidance.

### PR DESCRIPTION
**Pull request by deanwilson**
Once we have another team using it this should be updated to include
conftest.

### CODE REVIEW

File: source/standards/configuration-management.html.md.erb [hunk 1]
```diff
@@ -20,6 +20,27 @@ Use [Terraform][] to configure third party cloud infrastructure like Amazon Web
 
 Terraform supports a [large number of providers][], and you can configure it to support multiple environments with different parameters. See the [govuk-aws][] repository as an example.
 
+### Versioning
+
+Due to the high rate of change in many cloud provider offerings we
+recommend you keep your terraform versions and codebases up to date. A
+version manager such as [tfenv](https://github.com/tfutils/tfenv),
+already used by a number of GDS teams, can help you with supporting
+multiple versions.
```
Comment (issyl0) [inline source/standards/configuration-management.html.md.erb:29]: Is it worth mentioning `.terraform-version` files here too?

File: source/standards/configuration-management.html.md.erb [hunk 2]
```diff
@@ -20,6 +20,27 @@ Use [Terraform][] to configure third party cloud infrastructure like Amazon Web
 
 Terraform supports a [large number of providers][], and you can configure it to support multiple environments with different parameters. See the [govuk-aws][] repository as an example.
 
+### Versioning
+
+Due to the high rate of change in many cloud provider offerings we
+recommend you keep your terraform versions and codebases up to date. A
```
Comment (issyl0) [inline source/standards/configuration-management.html.md.erb:26]:
```suggestion
recommend you keep your Terraform versions and codebases up to date. A
```

File: source/standards/configuration-management.html.md.erb (hunk 1 above)
Comment (deanwilson) [reply source/standards/configuration-management.html.md.erb:29]: That's an implementation detail. Once they look at the tool they'll see it

File: source/standards/configuration-management.html.md.erb (hunk 2 above)
Comment (deanwilson) [reply source/standards/configuration-management.html.md.erb:26]: Fix incoming.

**Pull request merged**